.github/workflows/test.yml

```diff
@@ -56,18 +56,17 @@ jobs:
           ./gradlew check
           ./gradlew assemble
 
-      # There is no combination of flake8 and pyflakes versions which
-      # are compatible with the entire range of Python versions tested
-      # in our build matrix. Therefore, only run flake8 on the latest
-      # Python version:
-      - name: Run flake8
-        if: ${{ matrix.python-version == '3.9' }}
+      # Nose is no logner maintained, and does not support Python 3.10.
```

Comment:
> # Nose is no logner maintained
no longer

We could use pytest for all versions, but we will need an old release for 2.7 and 3.4 (4.6.9 I believe).